tests/format/js/arrows/with-inner-spacing/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -957,23 +1101,24 @@ export const bem = block =>
  * @param {String} block - the BEM Block you'd like to select.
  * @returns {Function}
  */
-export const bem = block =>
+export const bem =
+  block =>
   /**
    * @param {String} [element] - the BEM Element within that block; if undefined, selects the block itself.
    * @returns {Function}
    */
   element =>
-    /**
-     * @param {?String} [modifier] - the BEM Modifier for the Block or Element; if undefined, selects the Block or Element unmodified.
-     * @returns {String}
-     */
-    modifier =>
-      [
-        ".",
-        css( block ),
-        element ? \`__\${ css( element ) }\` : "",
-        modifier ? \`--\${ css( modifier ) }\` : "",
-      ].join( "" );
+  /**
+   * @param {?String} [modifier] - the BEM Modifier for the Block or Element; if undefined, selects the Block or Element unmodified.
+   * @returns {String}
+   */
+  modifier =>
+    [
+      ".",
+      css( block ),
```

Comment:
ditto: the indentation in this arrow function block also does not look right to me